src/utils/TextFieldValidation.js

```diff
@@ -4,6 +4,8 @@ export class TextFieldValidation {
         this.status = null;
         this.validationData = null;
     }
+
+    /* prevents input lag by delaying the run until typing ends */
```

Comment:
minor style note - these can be // comments and not /*/ comments because they're single-line comments. If they were multi-line comments, then I would expect /* */ around the comments.